### ISSUE TITLE
"Upgrades → Plans" Redirect Lacks Site Selection in Help Center

### ISSUE THREAD

**Issue by okmttdhr** [Automattic/wp-calypso#96391]
When users follow the "Upgrades → Plans" link within the ["Upgrade Your Plan" support document](https://wordpress.com/support/manage-purchases/upgrade-your-plan/) (accessed through the inline modal Help Center), they are redirected to the /plans page without selecting a specific site. This may confuse users who expect the selected site context to carry over.

**Steps to repro;**

- Open the Help Center inline modal within the application.
- Access the "Upgrade Your Plan" support document.
- Click on the Upgrades → Plans link.
- Observe that the redirect takes you to the /plans page, but no specific site is pre-selected.

**Suggested Solution:**

- Modify the link behavior so that the selected site context is preserved during redirection to /plans.

or

- Update the "Upgrade Your Plan" support documentation to inform users that the site may need to be selected manually upon 
- reaching the /plans page.

**Context:** p58i-iGA-p2

**Comment by Robertght** [comment]
I agree that it would be a great thing to have, and users should easily upgrade if needed.

@Automattic/vertex I'm flagging this one for you.

**Comment by agrullon95** [comment]
Added the domain to the URL for the current site when users click a `/plans` link within a support doc: https://github.com/Automattic/wp-calypso/pull/97861
